_posts/newsletters/2023-03-29-newsletter.md

```diff
@@ -0,0 +1,282 @@
+    
+[Learn more and register.](https://us-rse.org/ag/rse-gln)
+
+<a name="dei-speaker-series"></a>
+## US-RSE DEI-WG Speaker Series
+
+The US-RSE Diversity, Equity, and Inclusion Working Group (DEI-WG) is working to bring to the US-RSE community quarterly visits from RSE professionals with vested interests or research into DEI in the RSE community at large. Our community pledges to continually educate ourselves through research on best practices focused on diversity, equity, and inclusion. The US-RSE is committed to providing an inclusive environment with equitable treatment for all and to promoting and encouraging diversity throughout the RSE community. We encourage everyone to suggest speakers and topics and are open to different formats and timelines of the events. Feedback is much appreciated.
+
+<a name="dei-media-club"></a>
+## US-RSE DEI-WG Media Club
+
+The DEI Working Group hosts a regular Media Club discussion in which we invite participants to read, listen to, or watch the chosen DEI media and come ready to discuss the content as a small group. Next up is a discussion of this podcast on the Florida’s Department of Education rejecting an AP course on African American Studies, and what's happening in higher education there in general. [Listen to the podcast](https://voltedu.com/podcast/podcast-crossover-the-college-boards-trouble-in-florida/).
+
+[Link to register](https://stanford.zoom.us/meeting/register/tJwkce-uqjwoHdxLgipTO2ncD943cXWpmAeJ)
+
+<a name="community-calls"></a>
+## Community Calls
+    
+### April Community Call
+    
+The April community call will be on Friday, **April 14, 2pm ET/1pm CT/12pm MT/11am PT**.
+
+The topic of the community call will be the first ever US-RSE conference, which will take place in Chicago, IL, October 16-18, 2023. We are committed to making this event successful! To ensure its usefulness to the US-RSE community, we’d like to hear your thoughts and ideas! That’s why the next community call will be all about the conference. Are there specific events you would like to see? What would or wouldn’t make you attend? Do you have concrete ideas of how this conference could be useful to you? Join us and discuss with other members of the community.
+    
+**Registration details**
+You can register for this community call [here](https://asu.zoom.us/meeting/register/tZUkdOqrrDgqHdKXUbh__bHT-ENeSWMrdgLK).
+    
+### Community Call - Special Edition
+    
+On Thursday, **April 20 at 2pm ET/1pm CT/12pm MT/11am PT** we will host a special Community Call to outline upcoming plans for [our new grant](https://us-rse.org/2023-03-27-grant-announcement/) that US-RSE has received from The Alfred P. Sloan Foundation. This will provide an opportunity to answer any questions you may have. The call will be recorded and shared internally on Slack.
```

Comment:
```suggestion
On Thursday, **April 20 at 2pm ET/1pm CT/12pm MT/11am PT** we will host a special Community Call to outline upcoming plans for [the new grant](https://us-rse.org/2023-03-27-grant-announcement/) that US-RSE has received from the Alfred P. Sloan Foundation. This will provide an opportunity to answer any questions you may have. The call will be recorded and shared internally on Slack.
```